docs/porting/libraries.md

```diff
@@ -1,5 +1,153 @@
-# 🔧 Porting your libraries
+# Porting Libraries to .NET Core
 
-> **Note**
-> 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
+blah blah some intro
```

Comment:
I can do that.  I've been putting TODOs elsewhere.  I guess I should change my habit of putting "blah blah some intro" at the beginning of docs I write :smile: